app/models/orchestration_template_vnfd.rb

```diff
@@ -0,0 +1,65 @@
+class OrchestrationTemplateVnfd < OrchestrationTemplate
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+
+  before_create :raw_create, :if => :remote_proxy?
+  before_update :raw_update, :if => :remote_proxy?
+  before_destroy :raw_destroy, :if => :remote_proxy?
+
+  def raw_create
+    vnfd_data = {:attributes    => {:vnfd => content},
+                 :service_types => [{:service_type => "vnfd"}],
+                 :mgmt_driver   => "noop",
+                 :infra_driver  => "heat"}
+
+    vnfd_data[:name] = name unless name.blank?
+    vnfd_data[:description] = description unless description.blank?
+
+    connection_options = {:service => "NFV"}
+    ext_management_system.with_provider_connection(connection_options) do |service|
+      self.ems_ref = service.vnfds.create(:vnfd => vnfd_data, :auth => {}).id
+    end
```

Comment:
This four lines here feels very specific to Openstack Tacker.

Should this be refactored to a subclass within the provider namespace?